app/controllers/website_wizard_controller.rb

```diff
@@ -25,6 +25,8 @@ def update
         redirect_to next_wizard_path
         # render_wizard
       else
+        #notice with an active model error
+        flash[:notice] =  @website.errors.messages[:website].try(:first)
```

Comment:
I tried to get rid of that errors.add :base  and flash a message in the controller, but then our method  was always returning true and return false didn't help. I'll use the full messages method